cylc/flow/option_parsers.py

```diff
@@ -66,7 +73,7 @@ class CylcOptionParser(OptionParser):
     MULTITASKCYCLE_USAGE = MULTI_USAGE_TEMPLATE.format(
         "", WITH_CYCLE_GLOBS, "", WITH_CYCLE_EXAMPLES)
     MULTITASK_USAGE = MULTI_USAGE_TEMPLATE.format(
-        ", for a specific cycle point", WITHOUT_CYCLE_GLOBS,
+        ",\nfor a specific cycle point", WITHOUT_CYCLE_GLOBS,
```

Comment:
What was the motivation for adding in the newline? Now, for ``cylc insert``, the printed sentence is broken across lines, which looks wrong to me:

```
TASK_GLOB is a pattern to match tasks, or families, or groups of them, in
the task pool,
for a specific cycle point:
* CYCLE-POINT/TASK-NAME-GLOB
* CYCLE-POINT/FAMILY-NAME-GLOB
```

Reply:
Oops, I'll check this tomorrow.  Pretty sure I originally tried to split a line that was sticking out way further than all the others, but I may have screwed up.